### PR TITLE
UI improvement(Improved Logout Func)

### PR DESCRIPTION
**Pull request by Ankur9669**

### CODE REVIEW

File: FitVid/src/Components/NavBar/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export {
+  PrimaryButton,
+  Link,
+  useCategory,
+  useUser,
+  useHistory,
+  usePlayLists,
+  useWatchLater,
+  useLiked,
+  useToast,
```
Comment (manojsarna) [inline FitVid/src/Components/NavBar/index.js:20]: Good work on this , creating a separate index.js

File: FitVid/src/Reducer/history-reducer.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+    case "SET_HISTORY_LIST":
+      return [...action.payload.value];
+
+    case "RESET":
+      return [];
+
+    default:
+      return historyList;
+  }
+};
```
Comment (manojsarna) [inline FitVid/src/Reducer/history-reducer.js:12]: Handle exception cases for switch

**Review by manojsarna — APPROVED** [review]
overall code is good

Do check the comments

**Review by Adarsh-Patel-Dev — COMMENTED** [review]
If you are using hover then try to add some transition in styling for good UX 
e.g. `transition: 0.5s;`

Suggestion: Remove box shadow of login container as It's parent container already have box-shadow.
Rest UI seems good

File: FitVid/src/Reducer/history-reducer.js (hunk 2 above)
Comment (Ankur9669) [reply FitVid/src/Reducer/history-reducer.js:12]: I think it is handled here in default

**Pull request merged**